libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/roadname/api/MapboxRoadNameLabelApi.kt

```diff
@@ -1,25 +1,70 @@
 package com.mapbox.navigation.ui.maps.roadname.api
 
+import com.mapbox.api.directions.v5.models.BannerComponents
 import com.mapbox.navigation.base.road.model.Road
-import com.mapbox.navigation.ui.maps.roadname.RoadNameAction
-import com.mapbox.navigation.ui.maps.roadname.RoadNameProcessor
-import com.mapbox.navigation.ui.maps.roadname.RoadNameResult
 import com.mapbox.navigation.ui.maps.roadname.model.RoadLabel
+import com.mapbox.navigation.ui.shield.api.MapboxRouteShieldApi
+import com.mapbox.navigation.ui.shield.model.RouteShieldCallback
 
 /**
  * The API allows you to generate data to be rendered as a part of road name labels.
  */
-class MapboxRoadNameLabelApi {
+class MapboxRoadNameLabelApi constructor(
```

Comment:
This needs `@JvmOverload` to avoid breaking change.

Reply:
Resolved.